### DISCUSSION TITLE
Cambios del Web Service 2.0

### DISCUSSION THREAD

**Discussion by AndresRamos** [AndresRamos/ARSoftware.Cfdi.DescargaMasiva#4] [General]
Quiero usar este discussion para analizar e implementar los nuevos cambios del Web Service para la descarga masiva de CFDIs.

**Comment by AndresRamos** [comment]
Gracias a @yiyoMX por el pull request inicial #3 para implementar los nuevos cambios.

Al parecer cambios nuevos solamente son el la solicitud, haciendo cambio al atributo Receptor por el nodo "Receptores".

Los nuevos atributos en la solicitud son:

- TipoComprobante
- EstadoComprobante
- RfcACuentaTerceros
- Complemento
- UUID

Actualmente ya estan implementados en el branch de development https://github.com/AndresRamos/ARSoftware.Cfdi.DescargaMasiva/commits/development 

Tambien se libero la version 1.0.2-alpha en Nuget.
`Install-Package ARSoftware.Cfdi.DescargaMasiva -Version 1.0.2-alpha`

Voy a empezar a realizar las pruebas para validar que funciona la libreria y de ser asi ya estare liberando la nueva version a master.

**Reply by yiyoMX** [reply]
Muy bien, me parece una buena solución lo de la versión alpha como nuget.

**Comment by yiyoMX** [comment]
Ya lo probé y si funciona.

**Comment by AgustinLab** [comment]
Buen día Andres es normal que no se pueda descargar los xml, desde el 10 junio 2024 hice una solicitud alrededor de las 11 am me regreso el uuid que tengo que guardar para volver a realizar la solicitud de descarga, por lo que el estatus que me regresa es "solicitud es Aceptada. Mandar otra solicitud de verificación mas tarde para que el servicio web pueda procesar la solicitud" o sabrás si hay inconvenientes en el sat.
muchas gracias
